### ISSUE TITLE
help fails if oo contains a Sequence

### ISSUE THREAD

**Issue by jkyang92** [Macaulay2/M2#2333]
The following commands entered in a prompt cause an error in help
```m2
(1,2,3)
help ZZ -- You can put anything for ZZ
```
The error is
```
../../Macaulay2/packages/Macaulay2Doc/repl.m2:591:1:(2):[30]: error: no method installed for (1,2,3)
```
This seems similar to the issue in #963, but it's not quite the same bug.

**Comment by mahrud** [comment]
I can't reproduce this. Which version of M2 is this?

**Comment by jkyang92** [comment]
1.19, You have to run this in a prompt, not from a file.

**Comment by jkyang92** [comment]
So it's even more sensitive than that. You probably have to run it from a new prompt. So for me at least, it appears when I call `help ZZ` for the first time, it goes and actually builds the documentation in `Macaulay2Doc`. Then when it sees the `oo` variable as `(1,2,3)` it freaks out.

**Comment by DanGrayson** [comment]
I can't reproduce it, either.

**Comment by jkyang92** [comment]
I think figured out why I'm having this. I didn't run `ninja install-packages` and I'm running M2 from the build directory, and so it's generating the documentation on the fly, which then doesn't work because `oo` has already been defined. I don't know if this is a bug or just a limitation.

**Comment by DanGrayson** [comment]
Ah, here it is:
```m2
i2 : beginDocumentation()

i3 : errorDepth = 0

o3 = 0

i4 : foo=()

o4 = ()

o4 : Sequence

i5 : document { Key => symbol foo }
../../../../Macaulay2/m2/debugging.m2:23:6:(1):[12]: error: no method installed for ()
../../../../Macaulay2/m2/debugging.m2:23:6:(1):[12]: --entering debugger (type help to see debugger commands)
../../../../Macaulay2/m2/debugging.m2:23:6-23:27: --source code:
     olderror processArgs args)

ii6 : 
../../../../Macaulay2/m2/methods.m2:239:10:(1):[11]: --entering debugger (type help to see debugger commands)
../../../../Macaulay2/m2/methods.m2:239:10-239:53: --source code:
    else error("no method installed for ", toString key))

ii7 : 
../../../../Macaulay2/m2/document.m2:405:11:(1):[9]: --entering debugger (type help to see debugger commands)
../../../../Macaulay2/m2/document.m2:405:11-405:19: --source code:
     o := options s;

ii8 : 
../../../../Macaulay2/m2/document.m2:399:63:(1):[7]: --entering debugger (type help to see debugger commands)
../../../../Macaulay2/m2/document.m2:399:63-399:92: --source code:
getOptionDefaultValues Symbol   := x -> if value x =!= x then getOptionDefaultValues value x else emptyOptionTable
```

I think that line in document.m2 has to be beefed up so it doesn't try to get option default values unless the value of x is a method function.  Or maybe `getOptionDefaultValues` shouldn't even be called by `processUsage`.

This misbehavior is old.

**Comment by mahrud** [comment]
I see. I'll add a fix in #2335.
